server/methods/email.js

```diff
@@ -0,0 +1,17 @@
+Meteor.methods({
+  sendEmail: function sendEmail(template, to) {
+    Meteor.Mandrill.sendTemplate({
+      template_name: template,
+      template_content: [
```

Comment:
This line wont pass, as well as other properties, however this is how Mandrill needs them...